addons/website_sale/views/onboarding_views.xml

```diff
@@ -1,12 +1,16 @@
 <odoo>
     <!-- PAYMENT ACQUIRER -->
-    <template id="website_sale_onboarding_payment_acquirer_step" primary="True"
-              inherit_id="payment.onboarding_payment_acquirer_step">
-        <xpath expr="//t[@t-set='method']" position="replace">
-            <t t-set="method" t-value="'action_open_website_sale_onboarding_payment_acquirer'" />
-        </xpath>
-        <xpath expr="//t[@t-set='state']" position="replace">
-            <t t-set="state" t-value="state.get('website_sale_onboarding_payment_acquirer_state')" />
+    <template id="website_sale_onboarding_payment_acquirer_step" inherit_id="payment.onboarding_payment_acquirer_step">
+        <xpath expr="//t" position="replace">
+            <t t-call="base.onboarding_step">
+                <t t-set="title">Odoo Payments</t>
+                <t t-set="description">Allow your customers to pay online for their purchase.</t>
+                <t t-set="btn_text">Enable payments</t>
+                <t t-set="done_text">Payments enabled!</t>
```

Comment:
Doesn't consider the fact that the company might not have access to Odoo Payments (because in an unsupported country)

Reply:
Yes, so when Odoo Payments are not supported in the country of the current company, old labels are used.If it is supported, these new labels are used.